src/cli/common.rs

```diff
@@ -1,35 +1,29 @@
 use directories::UserDirs;
 use fs::OpenOptions;
 use git2::build::RepoBuilder;
-use git2::{Cred, FetchOptions, RemoteCallbacks};
-use std::env::temp_dir;
+use git2::{Cred, FetchOptions, RemoteCallbacks, Repository};
+use std::env::{current_dir, temp_dir};
 use std::fmt::Display;
 use std::fs;
 use std::io::{Error, ErrorKind, Write};
 use std::path::{Path, PathBuf};
 use toml::Table;
 use uuid::Uuid;
 
-pub const ARAKI_ENVS_DIR: &str = ".araki/envs";
 pub const ARAKI_BIN_DIR: &str = ".araki/bin";
-
-/// Get the user's araki envs directory, which by default
-/// is placed in their home directory
-pub fn get_default_araki_envs_dir() -> Option<PathBuf> {
-    let Some(araki_envs_dir) = UserDirs::new().map(|dirs| dirs.home_dir().join(ARAKI_ENVS_DIR))
-    else {
-        return UserDirs::new().map(|dirs| dirs.home_dir().join(ARAKI_ENVS_DIR));
-    };
-
-    if !araki_envs_dir.exists() {
-        println!(
-            "araki envs dir does not exist. Creating it at {:?}",
-            araki_envs_dir
-        );
-        let _ = fs::create_dir_all(araki_envs_dir);
+pub const ARAKI_GIT_DIR_NAME: &str = ".araki-git";
+
+pub fn get_araki_git_repo() -> Result<Repository, String> {
+    let cwd = current_dir().unwrap();
+    let araki_git_dir = cwd.join(ARAKI_GIT_DIR_NAME);
+    if !araki_git_dir.exists() {
+        return Err(format!(
+            "No araki git dir files found in {:?}",
+            araki_git_dir
+        ));
     }
-
-    UserDirs::new().map(|dirs| dirs.home_dir().join(ARAKI_ENVS_DIR))
+    let repo = Repository::open(araki_git_dir).expect("Failed to open repository");
```

Comment:
Let's propagate this error up, since the calling function returns a `Result` anyway.